src/Components/BookList.jsx

```diff
@@ -1,21 +1,25 @@
 import React, { Component } from 'react';
 
-export default class BookList extends Component {
-  constructor(props) {
-    super(props)
-  }
+const BookList = (props) => {
 
-  render () {
-    return (
-      <div id="bookList">
-        <ul>
-          <li></li>
-          <li></li>
-          <li></li>
-          <li></li>
-          <li></li>
-        </ul>
-      </div>
-    );
-  }
+  const books = props.books;
```

Comment:
you can use ES6 object destructuring here and it's a little cleaner:
```suggestion
  const { books } = props;
```